BITS-App/Views/ArticlePage.xaml.cs

```diff
@@ -1,11 +1,80 @@
+using HtmlAgilityPack;
+
 namespace BITS_App.Views;
 
 public partial class ArticlePage : ContentPage
 {
-	public ArticlePage()
-	{
-		InitializeComponent();
+    public ArticlePage()
+    {
+        InitializeComponent();
 
         BindingContext = new Models.Article(7767);
+
+        Dispatcher.Dispatch(async () => await LoadContentAsync());
     }
+
+    /// <summary>
+    /// Loads content to XAML from HTML Content binding of <see cref="Models.Article">Models.Article</see>
+    /// </summary>
+    public async Task LoadContentAsync()
```

Comment:
We're eventually going to need to wire this method into an event handler function, but this branch doesn't really have the event-listener toolkit right now, so it'll have to be done later.